locales/en/messages.json

```diff
@@ -1981,7 +1981,7 @@
     },
 
     "receiverHelp": {
-        "message": "Please read receiver chapter of the documentation.  Configure serial port (if required), receiver mode (serial/ppm/pwm), provider (for serial receivers), bind receiver, set channel map, configure channel endpoints/range on TX so that all channels go from ~1000 to ~2000.  Set midpoint (default 1500), trim channels to 1500, configure stick deadband, verify behaviour when TX is off or out of range.<br /><span class=\"message-negative\">IMPORTANT:</span> Before flying read failsafe chapter of documentation and configure failsafe."
+        "message": "<b>Basic setup</b>: For serial connections, go to the Ports tab and enable ‘Serial Rx’ for the UART to which the Rx is connected.  In the Receivers tab, choose ‘Serial based receiver…’, and set the correct link/data format at the ‘Serial Rx Provider’ drop-down.  For other types of receiver, set the correct mode (ppm, pwm etc).  Choose the correct ‘Channel Map’ for your radio.  Bind the receiver and check that the Roll, Pitch and other channel bar graphs move correctly when you move the sticks.  You may need to reconfigure Map, Mode, Channels or Mix so that, for example, moving the Roll stick changes Roll values.  Adjust the channel endpoint or range values in the transmitter so that all channels go from ~1000 to ~2000.  Trim the midpoint to 1500.  RC Smoothing is best left at defaults.  For more information, please read receiver chapter of the <strong><a href=\"https://github.com/betaflight/betaflight/blob/master/docs/Rx.md\" target=\"_blank\" rel=\"noopener noreferrer\">documentation</a></strong>.<br /><span class=\"message-negative\">IMPORTANT:</span>: Do not forget to configure and check failsafe in the Failsafe tab! OpenTx users MUST ensure that the ADC filter is un-checked (disabled) in the Hardware tab of their transmitter! Read the <strong><a href=\"https://github.com/betaflight/betaflight/wiki/Feed-Forward-2.0/#disable-adc-filter\" target=\"_blank\" rel=\"noopener noreferrer\">wiki</a></strong> for instructions."
```

Comment:
I think we should avoid to make false statements.
There is no reason that users cannot have the ADC filter enabled in OpenTX when using Betaflight, but depending on their settings they might get bad performance because of this. So if anything we should capture this in a correct statement like this:

'For performance reasons, it is highly recommended that users disable the ADC filter in OpenTX, for more information see...'

If we are adding links to an article, then this article should probably be moved into the documentation instead of being in wiki where it is subject to potentially incorrect user edits.

And I'd probably put this above the important advice to check the failsafe settings, or else make this into a list of important messages. Adding it as a second sentence will probably result in a lot of users overlooking it.

Also, it looks better if we use `<strong>` to emphasise instead of ALL CAPS.

Reply:
There is no situation where having the ADC active improves flight performance.  On defaults it always makes for significantly more motor noise.  The cause of this is unclear to the ordinary pilot.  With the auto-smoothing method being on by default, P and Feedforward get a lot of unnecessary spikes that cause inefficiency and motor overheating.  There is never any reason why any person flying Betaflight would want the ADC on.
Changing `MUST` to `should` is too soft, I think. People will tend to ignore a `should`.  
Perhaps `are STRONGLY recommended to` might be OK, but we need a strong indication here.